### DISCUSSION TITLE
캘린더

### DISCUSSION THREAD

**Discussion by lledellebell** [gather-around-and-code/study-js-deepdive#7] [General]
안녕하세요
@2Ju0 

![image](https://github.com/gather-around-and-code/study-js-deepdive/assets/48350491/3d4255a2-805f-42f7-880b-48c18c39365f)

기능 같은것들 위주로 뭔가 생각하고 싶은데 
또 막상 떠오르는 아이디어가 없네요 @.@ 
상상력이 부족한 J 라...
캘린더를 만들어보거나 알고리즘 문제를 풀어볼까요?

[leetcode] 412. Fizz Buzz
https://leetcode.com/problems/fizz-buzz/

**Comment by 2Ju0** [comment]
저도 한 번 생각해볼게요! 캘린더나 알고리즘 문제도 좋습니다😊
